rpcs3/Emu/CPU/CPUTranslator.h

```diff
@@ -4034,4 +4034,4 @@ llvm::CallInst* llvm_asm(
 #define LLVM_ASM_VOID(asm_, irb, ctx)\
 	llvm_asm(irb, asm_, {}, "", ctx)
 
-#endif
+#endif
```

Comment:
Why?

Reply:
No reason. I had changed it and reverted it later, it just was tagged as a change. 
Ignore changes to this file.

Reply:
It's missing a newline